commit 10caf0b2db3fc04fae24ca896cf21f6aed12bbbc
Author: Christian Hofstaedtler <ch@grml.org>
Date:   Fri Mar 25 23:02:38 2011 +0100

    support DNS in boot environment
    
    Thanks to Ulrich Dangel for discovering this. [Closes: issue848]

Index: live-boot-grml/backends/initramfs-tools/live.hook
===================================================================
--- live-boot-grml.orig/backends/initramfs-tools/live.hook	2012-12-17 21:01:56.339847815 +0100
+++ live-boot-grml/backends/initramfs-tools/live.hook	2012-12-17 21:02:16.468142747 +0100
@@ -227,23 +227,19 @@
 
 # Some experimental stuff
 
-case "${LIVE_DNS}" in
-	true)
-		[ "${QUIET}" ] || echo -n " dns"
-
-		# /lib/libnss_dns.so.*:a   DNS
-		# /lib/libnss_files.so.*:  /etc/hosts and /etc/passwd
-		# /lib/libnss_compat.so.*: /etc/passwd
-
-		for _SHLIB in $(find /lib -name 'libnss_dns.so.*')
-		do
-			copy_exec "${_SHLIB}"
-		done
-
-		# might be needed if /etc/hosts is used
-		#mkdir -p "${DESTDIR}/etc"
-		#cp -p /etc/nsswitch.conf "${DESTDIR}/etc"
-		;;
-esac
+[ "${QUIET}" ] || echo -n " dns"
+
+# /lib/libnss_dns.so.*:a   DNS
+# /lib/libnss_files.so.*:  /etc/hosts and /etc/passwd
+# /lib/libnss_compat.so.*: /etc/passwd
+
+for _SHLIB in $(find /lib -name 'libnss_dns.so.*')
+do
+	copy_exec "${_SHLIB}"
+done
+
+# might be needed if /etc/hosts is used
+#mkdir -p "${DESTDIR}/etc"
+#cp -p /etc/nsswitch.conf "${DESTDIR}/etc"
 
 [ "${QUIET}" ] || echo .
